docs/explanations/modeling_extensions/diagnostics/index.rst

```diff
@@ -2,9 +2,8 @@
 Degeneracy Hunter
 ==================================
 
-Degeneracy Hunter is a collection of tools for diagnostics of mathematical programs. The core ideas behind Degeneracy Hunter are explained here: https://www.sciencedirect.com/science/article/pii/B9780444635785501304
-
-Degeneracy Hunter is currently included in IDAES for beta testing purposes. This page will be updated as part of the official release. If you would like to help test Degeneracy Hunter, please look at the example notebook available here: https://github.com/IDAES/examples-pse/pull/21 Please report any thoughts, questions or bugs to: adowling@nd.edu
-
-Full documentation of the available methods can be :ref:`found here<reference_guides/core/util/model_diagnostics:Degeneracy Hunter>`.
+Degeneracy Hunter is a collection of tools for diagnostics of mathematical programs.
 
+* `Core ideas (conference paper) <https://www.sciencedirect.com/science/article/pii/B9780444635785501304>`_
+* `Example notebook <https://idaes.github.io/examples-pse/latest/Examples/Tools/degeneracy_hunter_doc.html>`_
```

Comment:
We will need to update this soon to point to the new examples repo, but other wise looks good.

@dangunter Any guidance on whether we should update the examples link now, or merge this as is and do another change later?